mdm-scripts/windows/scan.ps1

```diff
@@ -11,6 +11,8 @@
     Set 'cnspec' (default) to download from a share and execute it
     .PARAMETER RegistrationToken
     Is required to register the Mondoo Product, if ConfigFile is not existent
+    .PARAMETER ForceRegistration
+    Is required to force re-registration the Mondoo Product and overwrite the existing ConfigFile
```

Comment:
I would rephrase the description to explicitly state that the registered client will be logged out and re-registered